src/app/helpers/tests/jest-setup.js

```diff
@@ -1,3 +1,4 @@
 import fetch from 'jest-fetch-mock'; // eslint-disable-line import/no-extraneous-dependencies
 
 global.fetch = fetch;
+process.env.RAZZLE_ASSETS_MANIFEST = 'manifest.json';
```

Comment:
Is this the correct file to use? The notes state `Path to a file containing compiled asset outputs` and it defaults to `build/assets.json`. I'm not sure `manifest.json` is a correct replacement for this.

Reply:
`RAZZLE_ASSETS_MANIFEST` only exists when a bundle is created, which is not the case for when we unit test. The value that is set here is rather arbitrary, as ultimately we mock a virtual file https://github.com/bbc/simorgh/pull/470/files#diff-8228bd607779ed2f487609938246b1acR82.

Reply:
OK, since we're using an arbitrary value here, I'd suggest calling it `'mockBundleAssets.json'` just to clarify that the tests aren't actually using the contents of `'manifest.json'`.